### PR TITLE
Document environment variables of MacOS images

### PR DESCRIPTION
**Pull request by dsame**
# Description
Improvement

the following environment variables that are mentioned in documentation:

All platforms:
ANDROID_HOME
ANDROID_SDK_ROOT
ANDROID_NDK_HOME
CHROMEWEBDRIVER
EDGEWEBDRIVER
GECKOWEBDRIVER
VCPKG_INSTALLATION_ROOT
CONDA
ANT_HOME
GRADLE_HOME

## Check list
- [x] Related issue / work item is attached
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by AlenaSviridenko** [comment]
Android variable section is empty

File: images/macos/software-report/SoftwareReport.Browsers.psm1 [hunk 1]
```diff
@@ -49,4 +49,27 @@ function Get-FirefoxVersion {
 
 function Get-GeckodriverVersion {
     return Run-Command "geckodriver --version" | Select-Object -First 1
-}
\ No newline at end of file
+}
+
+function Build-BrowserWebdriversEnvironmentTable {
+    Write-Host "Build-BrowserWebdriversEnvironmentTable"
```
Comment (al-cheb) [inline images/macos/software-report/SoftwareReport.Browsers.psm1:55]: Why do ween need Write-Host?

File: images/macos/software-report/SoftwareReport.Common.psm1 [hunk 2]
```diff
@@ -472,4 +472,46 @@ function Get-SwiftLintVersion {
 function Get-PowershellVersion {
     $powershellVersion = Run-Command "powershell --version"
     return $powershellVersion
-}
\ No newline at end of file
+}
+
+function Build-PackageManagementEnvironmentTable {
+    Write-Host "Build-PackageManagementEnvironmentTable"
```
Comment (al-cheb) [inline images/macos/software-report/SoftwareReport.Common.psm1:478]: Why do ween need Write-Host?

File: images/macos/software-report/SoftwareReport.Generator.ps1 [hunk 3]
```diff
@@ -183,6 +190,9 @@ $markdown += New-MDList -Style Unordered -Lines ($lintersList | Sort-Object)
 
 $markdown += New-MDHeader "Browsers" -Level 3
 $markdown += Get-BrowserSection
+$markdown += New-MDHeader "Browsers environment variables" -Level 4
+$markdown += Build-BrowserWebdriversEnvironmentTable
```
Comment (al-cheb) [inline images/macos/software-report/SoftwareReport.Generator.ps1:194]: Should it be the table too?

File: images/macos/software-report/SoftwareReport.Common.psm1 [hunk 4]
```diff
@@ -472,4 +472,46 @@ function Get-SwiftLintVersion {
...
+    return @(
+        @{
+            "Name" = "VCPKG_INSTALLATION_ROOT"
```
Comment (al-cheb) [inline images/macos/software-report/SoftwareReport.Common.psm1:481]: Should we use alphabetical sort?

File: images/macos/helpers/SoftwareReport.Helpers.psm1 [hunk 5]
```diff
@@ -30,3 +30,23 @@ function New-MDNewLine {
     $newLineSymbol = [System.Environment]::NewLine
     return $newLineSymbol * $Count
 }
+
+function Get-LinkTarget {
+    param (
+        [string] $inputPath
+    )
+    $link = Get-Item $inputPath | Select-Object -ExpandProperty Target
+    If ($link) {
```
Comment (maxim-lobanov) [inline images/macos/helpers/SoftwareReport.Helpers.psm1:39]: styling: `if`, `else`

File: images/macos/software-report/SoftwareReport.Android.psm1 [hunk 6]
```diff
@@ -98,6 +100,18 @@ function Build-AndroidTable {
     }
 }
 
+function Build-AndroidEnvironmentTable {
+    Write-Host "Build-AndroidEnvironmentTable"
```
Comment (maxim-lobanov) [inline images/macos/software-report/SoftwareReport.Android.psm1:104]: Why do we need this Write-Host here?

File: images/macos/software-report/SoftwareReport.Android.psm1 [hunk 7]
```diff
@@ -98,6 +100,18 @@ function Build-AndroidTable {
...
+    $androidVersions = Get-Item env:ANDROID_*
```
Comment (maxim-lobanov) [inline images/macos/software-report/SoftwareReport.Android.psm1:105]: Please sort variables by name

File: images/macos/software-report/SoftwareReport.Browsers.psm1 [hunk 8]
```diff
@@ -49,4 +51,27 @@ function Get-FirefoxVersion {
...
+            "Value" = $env:EDGEWEBDRIVER
+        },
+        @{
+            "Name" = "GECKOWEBDRIVER"
+            "Value" = $env:GECKOWEBDRIVER
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-PathWithLink($_.Value)
```
Comment (maxim-lobanov) [inline images/macos/software-report/SoftwareReport.Browsers.psm1:74]: These variables are not symlinks so we don't need to use `Get-PathWithLink` here

File: images/macos/software-report/SoftwareReport.Common.psm1 [hunk 9]
```diff
@@ -472,4 +472,22 @@ function Get-SwiftLintVersion {
...
+            "Value" = $env:CONDA
+        },
+        @{
+            "Name" = "VCPKG_INSTALLATION_ROOT"
+            "Value" = $env:VCPKG_INSTALLATION_ROOT
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-PathWithLink($_.Value)
```
Comment (maxim-lobanov) [inline images/macos/software-report/SoftwareReport.Common.psm1:490]: These variables are not symlinks so we don't need to use `Get-PathWithLink` here

File: images/macos/software-report/SoftwareReport.Browsers.psm1 [hunk 10]
```diff
@@ -1,3 +1,5 @@
+Import-Module "$PSScriptRoot/../helpers/SoftwareReport.Helpers.psm1" -DisableNameChecking
```
Comment (maxim-lobanov) [inline images/macos/software-report/SoftwareReport.Browsers.psm1:1]: I don't think that we need this import here. `SoftwareReport.Helpers.psm1` in software generator file

**Review by maxim-lobanov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Comment by carstenhag** [comment]
Yay, thanks a lot!!

**Pull request merged**